### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by mpherman2**

### CODE REVIEW

File: prow/gerrit/adapter/adapter.go [hunk 1]
```diff
@@ -303,7 +303,7 @@ func (c *Controller) Sync() {
 		if len(changes) == 0 {
 			return
 		}
-		log.Infof("Processing %d changes for instance: %s", len(changes), instance)
+		log.WithFields(logrus.Fields{"Instance": instance, "Changes": len(changes)}).Infof("Finished querying instance for changes")
```
Comment (chaodaiG) [inline prow/gerrit/adapter/adapter.go:306]: So far all other logs are lowercase words connected by underscores I think
```suggestion
		log.WithFields(logrus.Fields{"instance": instance, "changes": len(changes)}).Infof("Finished querying instance for changes")
```

Comment (mpherman2) [reply prow/gerrit/adapter/adapter.go]: Done

**Review by chaodaiG — APPROVED** [review]

File: prow/gerrit/adapter/adapter.go [hunk 2]
```diff
@@ -303,7 +303,7 @@ func (c *Controller) Sync() {
 		if len(changes) == 0 {
 			return
 		}
-		log.Infof("Processing %d changes for instance: %s", len(changes), instance)
+		log.WithFields(logrus.Fields{"instance": instance, "changes": len(changes)}).Infof("Finished querying instance for changes")
```
Comment (chaodaiG) [inline prow/gerrit/adapter/adapter.go:306]:
```suggestion
		log.WithFields(logrus.Fields{"instance": instance, "changes": len(changes)}).Info("Finished querying instance for changes")
```
actually there is no format
/hold

**Review by chaodaiG — APPROVED** [review]
/unhold

**Pull request merged**